packages/nomad/clickhouse.hcl

```diff
@@ -56,14 +68,15 @@ job "clickhouse" {
         volumes = [
           "local/config.xml:/etc/clickhouse-server/config.d/gcs.xml",
           # disabled while testing but will pass password to orchestrator in the future
-          # "local/users.xml:/etc/clickhouse-server/users.d/users.xml",
+          "local/users.xml:/etc/clickhouse-server/users.d/users.xml",
         ]
       }
 
       template {
         data = <<EOF
 <?xml version="1.0"?>
 <clickhouse>
+    <shutdown_wait_unfinished_queries>1</shutdown_wait_unfinished_queries>
```

Comment:
makes clickhouse not exit until queries are finished

Reply:
you probably want to add 
```
<shutdown_wait_unfinished>60</shutdown_wait_unfinished>
```

Reply:
This field is a Boolean

Reply:
okay i got some clarification on this here [https://github.com/ClickHouse/ClickHouse/pull/77515](https://github.com/ClickHouse/ClickHouse/pull/77515)<br><br>TLDR:<br>we want this 

```
<clickhouse>
    <shutdown_wait_unfinished>30</shutdown_wait_unfinished>
    <shutdown_wait_unfinished_queries>1</shutdown_wait_unfinished_queries>
</clickhouse>
```